packages/firestore/package.json

```diff
@@ -49,16 +49,22 @@
   },
   "exports": {
     ".": {
-      "node": "./dist/index.node.cjs.js",
+      "node": {
+        "require": "./dist/index.node.cjs.js",
+        "import": "./dist/index.node.mjs"
```

Comment:
Where is the build rule for creating this file?

Reply:
It gets the name from whatever is in the `main-esm` field in package.json

Reply:
So here is the build config https://github.com/firebase/firebase-js-sdk/blob/2ad6ce8137c44bfc42cd4ea307e7c26c12fc065b/packages/firestore/rollup.config.js#L84

Reply:
I see. Did you test it in Nodejs - running `import {} from @firebase/firestore` ?

Reply:
I've added the workaround in Option 2 here: https://www.stefanjudis.com/snippets/how-to-import-json-files-in-es-modules-node-js/#option-2%3A-leverage-the-commonjs-%60require%60-function-to-load-json-files

It required some changes to tsconfig.json to avoid some TS compiler complaints but it compiles correctly and works in both a Node and a Sveltekit test project. It also works in both projects if I don't change tsconfig and I just `tsignore` the offending lines (`import module from 'module'` and `const require = module.createRequire(import.meta.url);`) so any thoughts on which way is better?

Reply:
I'd use `ts-ignore` with comments here because we don't want these changes to affect browser code which shares the same tsconfig with Node code.